x-pack/legacy/plugins/index_management/public/app/components/mappings_editor/constants/field_options.tsx

```diff
@@ -154,4 +154,75 @@ export const PARAMETERS_OPTIONS: {
       ...getOptionTexts('termVector.withPositionsOffsetsPayloads'),
     },
   ] as SuperSelectOption[],
+  orientation: [
```

Comment:
We can leave it for now, but in my PR I used a Super select with description for those 2 options.

Reply:
updated to use super select